src/components/views/CourseScheduler/Course.jsx

```diff
@@ -208,19 +208,22 @@ const Course = ({
   };
 
   const semesterInNumber = () => {
+    // this is currently only stable for 2024 Winter/Spring (NOT 2023 Fall, which is part of same academic year)
+    // this controls the termID flag in the HTTP request, which Follett changed recently and may change again
+    const yearVal = 100080547 + course.year;
     switch (course.semester) {
       case "Fall":
-        return 1001 + 10 * (course.year % 100);
+        return yearVal - 2; // this does not currently work!
       case "Winter":
-        return 1002 + 10 * (course.year % 100);
+        return yearVal;
       default:
-        return 1003 + 10 * (course.year % 100);
+        return yearVal + 1;
     }
   };
```

Comment:
I think this field may no longer be formatted at all... Might just be a UUID.

Here are some examples that may persuade you:

| Semester | Institution | `termId` field | sample link |
|--|--|--|--|
| 2024 Winter | Williams |100082571 | [link](https://www.bkstr.com/williamsstore/course-materials-results?shopBy=course&divisionDisplayName=&departmentDisplayName=PHIL&courseDisplayName=14&sectionDisplayName=01&programId=912&termId=100082571) |
| 2024 Winter | Stanford |100080358 | [link](https://www.bkstr.com/stanfordstore/course-materials-results?shopBy=course&divisionDisplayName=Stanford&departmentDisplayName=PHIL&courseDisplayName=1&sectionDisplayName=01&programId=562&termId=100080358) |
| 2024 Winter | NYU |100082037 | [link](https://www.bkstr.com/nyustore/course-materials-results?shopBy=course&divisionDisplayName=&departmentDisplayName=ANTH-UA&courseDisplayName=319&sectionDisplayName=001&programId=4582&termId=100082037) |
| 2024 Spring | Williams |100082572 | [link](https://www.bkstr.com/williamsstore/course-materials-results?shopBy=course&divisionDisplayName=&programId=912&departmentDisplayName=PHIL&courseDisplayName=225&sectionDisplayName=T1&termId=100082572) |
| 2024 Spring | Pomona |100081752 | [link](https://www.bkstr.com/claremontstore/course-materials-results?shopBy=course&divisionDisplayName=POMONA%20COLLEGE&departmentDisplayName=PHIL&courseDisplayName=030&sectionDisplayName=01&programId=1550&termId=100081752) |

IMO there are no rules to them at all...

Reply:
Agreed, definitely a UUID. I'll delete the `switch` because nobody is getting textbooks for winter study anyway, so then currently this only works for spring. I wasn't able to find the Fall UUID either.